### DISCUSSION TITLE
Any ETA on 1.21.2 support?

### DISCUSSION THREAD

**Discussion by kevinwlee787** [Faithful-Resource-Pack/Faithful-Java-64x#29] [General]
Just wondering what the timeline is on 1.21.2 support? (Yes I know it came out yesterday)

**Comment by DMgaming100** [comment]
a possible 64x release is planned for later this week

**Reply by kevinwlee787** [reply]
Sweet, you guys are my heroes

**Comment by HARYA301** [comment]
What's up! The update is planned to release this Sunday.
Have a great day!
